docs/UserGuide.md

```diff
@@ -367,23 +370,24 @@ A quick overview of all supported commands, their formats and examples are given
 
 | Term                                         | Description                                               |
 | -------------------------------------------- | --------------------------------------------------------- |
-| **API (Application Programming Interface)** | An interface for two systems to interact with each other  |
-| **Endpoint** | The point of entry in a communication channel for two systems to interact with each other |
-| **Request** | A process in which information is sent out to an endpoint through one of the [request methods](#83-request-methods) |
-| **Response** | The information obtained from an endpoint after a request is sent to it (commonly in the [JSON format](#84-json-format)) |
+| **API** | API is short for **Application Programming Interface** and allows two systems to interact with each other  |
+| **Call** | A call to an API endpoint refers to the process of sending a [request](#83-what-are-requests-and-responses) to the server           |
+| **Endpoint** | The communication point of a system that allows it to interact with another system, commonly accessed through a URL |
+| **Request** | A process in which information is sent out to an endpoint through one of the [request methods](#84-request-methods) (a more detailed explanation can be found [here](#83-what-are-requests-and-responses)) |
+| **Response** | The information obtained from an endpoint after a request is sent to it (a more detailed explanation can be found [here](#83-what-are-requests-and-responses)) |
 | **Parameter**   | Information passed in as part of a command with its type identified by a prefix (e.g. <span class="compulsory-param">METHOD</span>) |
 | **Prefix**   | Characters used to identify the following parameter (e.g. <span class="compulsory-param">-x</span> is the prefix for the parameter <span class="compulsory-param">METHOD</span>) |
-| **JSON (JavaScript Object Notation)** | A lightweight format for data storage (a more detailed explanation can be found [here](#84-json-format)) |
-| **CURL (Client URL)** | A command-line tool used in the transfer of data via different network protocols |
-| **Index** | Index in this guide refers to the position of the endpoint in the endpoint list. It is signified by the number beside the endpoint. |
+| **JSON** | JSON is short for **JavaScript Object Notation** which is a lightweight format for data storage (a more detailed explanation can be found [here](#85-json-format)) |
+| **CURL** | CURL is short for **Client URL** and is a command-line tool used in the transfer of data via different network protocols |
```

Comment:
minor typo? should be `and it is`

Reply:
I am outside so can't update this now. I'll merge first so anyone hoping to work on the UG can go ahead and then fix the typo in another PR :)